mobile/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
The boil plate is auto generate by running "tsc --init". Let me know if any commented line is not needed so I could take them out.